client/app/profile/profile.controller.js

```diff
@@ -22,6 +22,8 @@
         vm.errorSetRole = false;
         vm.errorSetLevel = false;
         vm.errorSetContactDetails = false;
+        vm.verificationEmailSent = false;
+        vm.errorVerificationEmailSent = false;
```

Comment:
It is needed because when the user updates their Facebook, Twitter or LinkedIn details, but not their email address then no email is sent but it was not an error either so can't use the same variable for both cases.